### ISSUE TITLE
Access to GitHub.com organizations that use SSO

### ISSUE THREAD

**Issue by lbrownell-gpsw** [microsoft/appcenter#556]
**Describe the solution you'd like**
Allow GitHub Enterprise Cloud users who use SSO/SAML for GitHub access to use App Center.

**Describe alternatives you've considered**
Not using App Center.

**Additional context**
GitHub (a Microsoft company!) has two Enterprise offerings; this one (Enterprise Cloud) is hosted at https://github.com, and implements private repos as do the lower-priced offerings. It also allows the use of SSO/SAML for secure authentication (we also use 2FA, if that's an issue). Because of this, App Center can't see any of our repos in our organizations that are in Enterprise (we have one that isn't, and we can see all its repos); it doesn't even show that those organizations exist, even though at least one has some public repos.

There are two ways to gain access: Use https for repo access, but use a PAT instead of a password, or use git/ssh and use an SSH key that has been enabled for SSO in the GitHub organization.  Perhaps there is a third way to do it via the access request to GitHub you do now.

As it stands, this makes building and testing a non-starter for us.

**Comment by nrajpurkar** [comment]
@lbrownell-gpsw Thanks for bringing up this request. This is not possible today, but we'll track at as an improvement for the future.

**Comment by lbrownell-gpsw** [comment]
I TAKE IT BACK!  It does work.  

When I first enabled github, I clicked on the "accept" button (I think it is) for the specific Enterprise/2FA/SSO repo...and a spinner icon appeared and never went away.  After a long time, I hit the submit button, and, as reported, no access to our Enterprise repos.

After talking to support, I tried it again -- this time, it worked immediately, there were the repos, and I was able to build using one.

Thanks, sorry, closing this.
